scripts/generate_json.py

```diff
@@ -0,0 +1,87 @@
+from __future__ import unicode_literals
+from gym import envs, spaces 
+import json
+import os
+import sys
+import hashlib
+
+import logging
+logger = logging.getLogger(__name__)
+
+from gym.envs.tests.test_envs import should_skip_env_spec_for_tests
+from gym.envs.tests.test_envs_semantics import generate_rollout_hash, hash_object
+
+DATA_DIR = os.path.join(os.pardir, 'gym', 'envs', 'tests')
```

Comment:
nit: I would construct the path relative to __file__ here. Otherwise if you run the script from a different directory you won't find the data file.